### DISCUSSION TITLE
How make frontend part in the same app with orchid ?

### DISCUSSION THREAD

**Discussion by sergeynilov** [orchidsoftware/platform#2434] [Q&A]
Making in laravel/framework 9.1 admin area with orchid/crud 3.8 and orchid/platform 13.6
at some time it was decided to make frontend part in the same app, where we can have 
pages without “/admin” prefix in url and not logged user can open this page.

If there is a way to do it ? If yes, please reference to example...

Thanks!

**Comment by tabuna** [comment]
Hi @sergeynilov. You only need to declare screen routes where no authorization is required. 
The easiest way to specify them in `routes/web.php`

**Reply by sergeynilov** [reply]
Thanks! It works in this way. Also if there is a way to overwrite exsting current admin layout (left sidebase, top menu )?
